tools/terser.d.ts

```diff
@@ -145,7 +145,7 @@ export interface MinifyOutput {
     ast?: AST_Node;
     code?: string;
     error?: Error;
-    map?: string;
+    map?: any;
```

Comment:
Regarding the typing, didn't want to set the type as `any`. I recalled`recast` uses `any` for it's source-map

https://github.com/benjamn/recast/blob/edfae82bdb45ca736580bc951b4ef52cd363383b/lib/printer.ts#L13-L16

Reply:
I think it could be typed `SourceMapGenerator | string`. Any comments?

Reply:
I think correct type is `RawSourceMap | string`